spec/models/user_spec.rb

```diff
@@ -1,5 +1,54 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'tutor' do
+    before(:each) do
+      @user = User.new
+    end
+
+    it 'ensures tutors have role 0' do
```

Comment:
I think this test is more testing the default value of new users with unspecified permission roles?